bee-protocol/src/workers/mod.rs

```diff
@@ -58,9 +61,13 @@ where
     node_builder
         .with_worker::<MetricsWorker>()
         .with_worker::<PeerManagerResWorker>()
-        .with_worker_cfg::<PeerManagerWorker>(events)
+        .with_worker_cfg::<PeerManagerWorker>(PeerManagerConfig {
+            network_rx: network_events,
+            peering_rx: autopeering_events,
+            network_name: network_id.0,
+        })
         .with_worker_cfg::<HasherWorker>(config.clone())
-        .with_worker_cfg::<ProcessorWorker>(network_id)
+        .with_worker_cfg::<ProcessorWorker>(network_id.1)
```

Comment:
See comment above, it's a little bit hard to see what is going on here.

Reply:
I agree with you. This should become much more readable when we have introduce the `NetworkSpec` struct I mentioned above.